django-stubs/contrib/auth/base_user.pyi

```diff
@@ -32,6 +32,7 @@ class AbstractBaseUser(models.Model):
     def set_unusable_password(self) -> None: ...
     def has_usable_password(self) -> bool: ...
     def get_session_auth_hash(self) -> str: ...
+    def get_session_auth_fallback_hash(self) -> str: ...
```

Comment:
I believe this should also be `Iterable[str]` https://github.com/django/django/blob/c499184f198df8deb8b5f7282b679babef8384ff/django/contrib/auth/base_user.py#L138-L140

Reply:
https://github.com/typeddjango/django-stubs/pull/2407

Reply:
thanks :)